simtools/data_model/meta_data_model.py

```diff
@@ -0,0 +1,307 @@
+"""
+Definition of metadata model for input to and output of gammasim-tools.
+Follows CTA top-level data model definition.
+Metadata is broken into hierarchical categories in this module.
+
+Metadata definitions require the three following fields to be defined:
+
+* type: data type (str)
```

Comment:
Is the `(str)` an example? I.e., can it be changed to be `(str, int, float, email, etc.)`? Or does it have to be a string? It's a bit confusing to me.

Reply:
I removed as it is obviously confusing: with `str` it was meant the that data type is given as string (e.g. "float"). I don't think this is needed here, so removed it completely.